packages/types/mongoose-store/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as Promise from 'bluebird';
+import * as mongoose from 'mongoose';
+
+export default class MongooseStore<T>{
+  constructor(protected conn: mongoose.Connection) {
+
+  }
+}
+
+type ID = string | number;
```

Comment:
`ObjectId` is a mongoose object which can be serialized to a string. If we are going down the route of having a single, replaceable store module, I think it might be reasonable to require an id in these native data structures.

We'd be using them in REST api endpoints for a single resource and so on, so an identifier that can be stringifyed is a must.